crates/wysiwyg/src/composer_model.rs

```diff
@@ -101,29 +104,38 @@ where
         start: usize,
         end: usize,
     ) -> ComposerUpdate<C> {
-        let range = self.state.dom.find_range(start, end);
+        self.do_replace_text_in_range(
+            new_text,
+            start,
+            self.state.dom.find_range(start, end),
+        )
+    }
+
+    fn do_replace_text_in_range(
+        &mut self,
+        new_text: &[C],
+        mut start: usize,
+        range: Range,
+    ) -> ComposerUpdate<C> {
```

Comment:
It's not really an issue, but couldn't this still be used as a single function passing start, end and calculating the range inside?

Reply:
Thank you - that was left around from me using fake ranges. Fixed in d1faf9f